Scripts/DCS-BIOS/lib/common/Functions.lua

```diff
@@ -0,0 +1,7 @@
+
+---@func Returns path of scripts calling it
+---@return string
+function ScriptPath()
+    local str = debug.getinfo(2, "S").source:sub(2)
+    return str:match("(.*/)")
+ end
```

Comment:
It's best to end a file with a trailing newline. That way if someone else has to come in here later and make changes, it doesn't show that they were the last one to change this line.

Reply:
I always forget, yes you told me before. Maybe now it will stick.